mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/json/HeaderCollection.java

```diff
@@ -81,7 +82,7 @@ private Builder() {
 
         public Builder add(Field field) {
             Preconditions.checkNotNull(field);
-            String headerName = field.getName().toLowerCase();
+            String headerName = field.getName().toLowerCase(Locale.US);
```

Comment:
Could we define Locale.US as default locale some where very common then it's easier to change the default locale later on?